package.json

```diff
@@ -3,23 +3,40 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "lint": "eslint --ignore-path .gitignore .",
-    "start": "node ./bin/www"
+    "coverage": "nyc mocha --recursive --exit",
```

Comment:
Planning to use nyc to report test coverage with Circle or CodeShip and CodeClimate in another PR... maybe autoships to heroku?

Reply:
adding Issue